package.json

```
@@ -45,6 +45,7 @@
     "install:docs": "(cd docs && npm install --legacy-peer-deps)",
     "lint": "eslint '**/*.{js,ts,tsx,md,mdx}' --max-warnings=0",
     "lint:fix": "npm run lint -- --fix",
+    "markdownlint": "markdownlint-cli2 \"**/*.{md,mdx}\" \"!.github\" \"!.changeset\" \"!node_modules\" \"!CHANGELOG.md\"",
```

We have `node_modules` in a few more places such as `docs/` so do you think would it makes sense to extend the coverage like this?
 
```suggestion
    "markdownlint": "markdownlint-cli2 \"**/*.{md,mdx}\" \"!.github\" \"!.changeset\" \"!**/node_modules\" \"!CHANGELOG.md\"",
```